### ISSUE TITLE
Harbor Helm Push Fails with internal error

### ISSUE THREAD

**Issue by sidewinder12s** [goharbor/harbor#15347]
**Expected behavior and actual behavior:**

Helm Pushes to Chart Museum should work.

Helm Pushes to ChartMuseum fail with:

```log
Error: 500: could not properly parse response JSON: {"errors":[{"code":"UNKNOWN","message":"internal server error"}]}
```

I finally found useful errors in the logs, which seem to point possibly towards some DB issues with OIDC users similar to #15241. But I am unsure as I have not had issues logging into Harbor through OIDC, just using Helm to push to chartmuseum. 

Pushes and pulls from Docker (And pushes to the OCI Helm Registry) still work as well.

**Steps to reproduce the problem:**

1. Wait ~1 day after your last login to Harbor. 
2. Try to helm push to a ChartMuseum repo
3. Failure
4. Login to Harbor UI with OIDC again
5. Push again 
6. The push is successful

Logs from Harbor Core are attached

**Versions:**
Please specify the versions of following systems.

- harbor version: 2.3.0

[helm-push-error.csv](https://github.com/goharbor/harbor/files/6864132/helm-push-error.csv)

**Comment by sidewinder12s** [comment]
I have not tested with 2.3.1 but as soon as we upgrade was going to retest.

**Comment by reasonerjt** [comment]
@sidewinder12s Thanks, and to confirm it's caused by #15241 please refer to the comments in https://github.com/goharbor/harbor/issues/15294

Please also let us know if it's fixed in v2.3.1

**Comment by sidewinder12s** [comment]
Yup appears it is fixed in v2.3.1
